client/my-sites/site-settings/manage-connection/index.jsx

```diff
@@ -15,39 +14,21 @@ import DocumentHead from 'components/data/document-head';
 import HeaderCake from 'components/header-cake';
 import Main from 'components/main';
 import SiteOwnership from './site-ownership';
-import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
-import { isJetpackSite } from 'state/sites/selectors';
-import { isSiteAutomatedTransfer } from 'state/selectors';
+import redirectNonJetpack from 'my-sites/site-settings/redirect-to';
```

Comment:
Nit: For consistency I'd recommend to name the import the same way you named the component when you exported it.